zubhub_backend/zubhub/activities/tests/test_utils.py

```diff
@@ -0,0 +1,223 @@
+from django.test import TestCase
+from activities.models import * #import all the models
+from activities.utils import * #import all the utilities
+
+class TestUtils(TestCase):
+  def setUp(self):
```

Comment:
function name in python are preferable when in snake casing and not Camel Casing or lower camel casing like in your case.

Reply:
For the most part, yes. However, note [this](https://docs.djangoproject.com/en/5.0/topics/testing/overview/#writing-tests) example in the **official** Django documentation website.